services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx

```diff
@@ -23,6 +23,21 @@ import { Application } from '../../../api/api';
 export const ServiceLane: React.FC<{ application: Application }> = (props) => {
     const { application } = props;
     const releases = useDeployedReleases(application.name);
+    const releases_lane = releases.map((rel, index) => {
```

Comment:
```suggestion
    const releases_lane = !!releases && releases.map((rel, index) => {
```
just to make sure, Kuberpult will not crash if there is no releases